src/VirtualClient/VirtualClient.Core/VirtualClientComponentExtensions.cs

```diff
@@ -476,7 +476,7 @@ public static bool TryGetPackageStoreManager(this VirtualClientComponent compone
              * 
              * The following characters are not allowed: " \ / : | < > * ?
              * Directory and file names are case-preserving and case-insensitive.
-             * A path name may be no more than 2,048 characters in length. Individual components in the path can be a maximum of 255 characters in length.
+             * A path name may be no more than 2,048 characters in length. Individual components in the path can be a maximum of 255 --time=1800 characters in length.
```

Comment:
Make sure this --time is intended and it's not a result of some other replacement you were doing. If so, check all the replacement and make sure you didn't accidentally replaced other things